boards/m5stack/m5stack_cores3/m5stack_cores3_procpu_common.dtsi

```diff
@@ -19,6 +19,7 @@
 		zephyr,code-partition = &slot0_partition;
 		zephyr,rtc = &bm8563_rtc;
 		zephyr,bt-hci = &esp32_bt_hci;
+		zephyr,touch = &ft6336_touch;
```

Comment:
`ft6336` looks to me like a typo. Driver is called `ft5336`.

Reply:
It is not a typo. FT6336 is a variant and can be used with ft5663 driver:
```
description: FT3267/FT5XX6/FT6XX6 capacitive touch panels

compatible: "focaltech,ft5336"
```

Reply:
ok, I didn't know that. Thx for clarifying this!